cmd/crank/pkg/pkg.go

```diff
@@ -16,39 +16,34 @@ limitations under the License.
 
 package pkg
 
-// Cmd is the root package command.
-type Cmd struct {
-	Build BuildCmd `cmd:"" help:"Build a Crossplane package."`
-	Push  PushCmd  `cmd:"" help:"Push a Crossplane package to a registry."`
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+)
+
+// CrossplanePackage is a minimal struct to grab the name field out of the crossplane.yaml file
```

Comment:
I think it may be a good idea to move the contents of this file to top-level `pkg/` but that can be follow-up :+1: